types/trace-engine.d.ts

```diff
@@ -0,0 +1,1516 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import Protocol from 'devtools-protocol';
+import {TraceEvent} from './artifacts.ts';
```

Comment:
```suggestion
import {Protocol} from 'devtools-protocol';
import {TraceEvent} from './artifacts.js';
```

This fixed the type checking errors on my end.